ods_ci/run_interop.sh

```diff
@@ -8,6 +8,26 @@ export ROBOT_EXTRA_ARGS="-i Smoke --dryrun"
 TEST_CASE_FILE="tests/Tests"
 TEST_VARIABLES_FILE="test-variables.yml"
 
+if [[ -z "${TEST_SUITE}" ]]; then
+  echo "Error: TEST_SUITE not set. Please define it. Exiting.."
+  exit 1
+fi
+
+if [[ -z "${ARTIFACT_DIR}" ]]; then
+  echo "ARTIFACT_DIR is empty. Using  default value /tmp."
+  ARTIFACT_DIR="/tmp"
+fi
+
+if [[ ${TEST_SUITE} == "Post-Upgrade" ]]; then
+  echo "Retrive test config file..."
+  cp ${SHARED_DIR}/test-variables.yml test-variables.yml
+
+  echo "Running post-upgrade testing"
+  TEST_CASE_FILE="tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot"
```

Comment:
I think it will be better to use the tag `PostUpgrade` 
for more details, you can see the comment below https://github.com/red-hat-data-services/ods-ci/pull/2213/files#diff-2d80a284294fa1f65798ca0f1558ac396aabd2075ea029de95b2c23a40d961cdR93